package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "adopt-a-family",
+  "version": "0.1.0",
+  "description": "basic routes",
+  "main": "app.js",
+  "directories": {
+    "test": "test"
```

Comment:
Is this still accurate? Should the test folder also include `src/__test__` ?

Reply:
Upon further review, this seems to be only needed if we publish to npm. We should/could remove this? Thoughts?